larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -566,7 +584,7 @@ StatusCode MasterAlgorithm::RunSliceReconstruction(SliceVector &sliceVector, Sli
 
     if (m_shouldRunNeutrinoRecoOption && m_shouldRunCosmicRecoOption && (nuSliceHypotheses.size() != crSliceHypotheses.size()))
         throw StatusCodeException(STATUS_CODE_INVALID_PARAMETER);
-
+    
```

Comment:
Will need to run the removeWhitespace script over all changed files before can submit a PR to LArSoft/larpandoracontent